bitacross-worker/bitacross/core/bc-enclave-registry/src/lib.rs

```diff
@@ -251,4 +258,11 @@ impl EnclaveRegistryLookup for EnclaveRegistry {
 		let registry = self.registry.read().unwrap();
 		registry.iter().map(|(k, v)| (k.clone(), v.clone())).collect()
 	}
+
+	#[cfg(feature = "sgx")]
```

Comment:
A stupid question: why do we have `sgx` and `std` separately? They seem identical - do I overlook anything

Reply:
Thanks!

Usually we have different impls - but in this case they are identical so I've merged them